### ISSUE TITLE
where is the CPR corpus named CP-17?

### ISSUE THREAD

**Issue by CV2014** [jbjorne/TEES#33]
Only BioNLP09-13 and DDI 11-13 can be found, where is the CP17 training set and  test set?

**Comment by jbjorne** [comment]
The CP17 corpus can be automatically downloaded and converted with TEES, but since this corpus isn't published with an open license, it cannot be included in the installer. To download and convert the corpus from the original files, you can use the preprocessor command:

`python preprocess.py -s CONVERT_CHEMPROT,DIVIDE_SETS -o /tmp/CP17`

If you want to do the rest of the TEES preprocessing (sentence splitting, parsing etc.) you can combine this with the default preprocessing commands:

`python preprocess.py -s CONVERT_CHEMPROT,GENIA_SPLITTER,BLLIP_BIO,STANFORD_CONVERT,SPLIT_NAMES,FIND_HEADS,DIVIDE_SETS -o /tmp/CP17`
